fenix/app/src/androidTest/java/org/mozilla/fenix/ui/CollectionTest.kt

```diff
@@ -94,6 +95,18 @@ class CollectionTest {
         }
     }
 
+    @Test
```

Comment:
Are you adding the TestRail links here and for the other tests below?

Reply:
I thought we could add them when we get to that class. Just in case they get modified again.

Reply:
Done.